Xamarin.Forms.Platform.Android/CollectionView/ItemsViewRenderer.cs

```diff
@@ -139,7 +139,13 @@ protected override void Dispose(bool disposing)
 				if (Element != null)
 				{
 					TearDownOldElement(Element as ItemsView);
+				}
 
+				// Dispose the adapter after tear down element
```

Comment:
So this is a duplicate of the fix in #6467? In that case, remove the fix from that PR and we'll just review it separately. 

Also, see my comments about the fix in that PR - I don't think this actually fixes the problem.

Reply:
Yes I have made it after #6467, so the PR can be merged faster.
I will remove this part form the other and recheck this particular change.